altair/vegalite/v5/schema/_config.py

```diff
@@ -0,0 +1,7078 @@
+    days
+        The full names of the weekdays, starting with Sunday.
+    months
+        The full names of the months (starting with January).
+    periods
+        The A.M. and P.M. equivalents (e.g., ["AM", "PM"]).
+    shortDays
+        The abbreviated names of the weekdays, starting with Sunday.
+    shortMonths
+        The abbreviated names of the months (starting with January).
+    time
+        The time (%X) format specifier (e.g., "%H:%M:%S").
+    """
+
+    date: str
+    dateTime: str
+    days: Sequence[str]
+    months: Sequence[str]
+    periods: Sequence[str]
+    shortDays: Sequence[str]
+    shortMonths: Sequence[str]
+    time: str
+
+
+class TitleConfigKwds(TypedDict, total=False):
+    """
+    TitleConfig ``TypedDict`` wrapper.
+
+    Parameters
+    ----------
+    align
+        Horizontal text alignment for title text. One of ``"left"``, ``"center"``, or
+        ``"right"``.
+    anchor
+        The anchor position for placing the title and subtitle text. One of ``"start"``,
+        ``"middle"``, or ``"end"``. For example, with an orientation of top these anchor
+        positions map to a left-, center-, or right-aligned title.
+    angle
+        Angle in degrees of title and subtitle text.
+    aria
+        A boolean flag indicating if `ARIA attributes
+        <https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA>`__ should be
+        included (SVG output only). If ``false``, the "aria-hidden" attribute will be set on
+        the output SVG group, removing the title from the ARIA accessibility tree.
+
+        **Default value:** ``true``
+    baseline
+        Vertical text baseline for title and subtitle text. One of ``"alphabetic"``
+        (default), ``"top"``, ``"middle"``, ``"bottom"``, ``"line-top"``, or
+        ``"line-bottom"``. The ``"line-top"`` and ``"line-bottom"`` values operate similarly
+        to ``"top"`` and ``"bottom"``, but are calculated relative to the *lineHeight*
+        rather than *fontSize* alone.
+    color
+        Text color for title text.
+    dx
+        Delta offset for title and subtitle text x-coordinate.
+    dy
+        Delta offset for title and subtitle text y-coordinate.
+    font
+        Font name for title text.
+    fontSize
+        Font size in pixels for title text.
+    fontStyle
+        Font style for title text.
+    fontWeight
+        Font weight for title text. This can be either a string (e.g ``"bold"``,
+        ``"normal"``) or a number (``100``, ``200``, ``300``, ..., ``900`` where
+        ``"normal"`` = ``400`` and ``"bold"`` = ``700``).
+    frame
+        The reference frame for the anchor position, one of ``"bounds"`` (to anchor relative
+        to the full bounding box) or ``"group"`` (to anchor relative to the group width or
+        height).
+    limit
+        The maximum allowed length in pixels of title and subtitle text.
+    lineHeight
+        Line height in pixels for multi-line tit
```

Comment:
`padding` is an interesting edge case. 

The definition is a union that contains an inline `dict`.

Currently the `"googlecharts"` theme fails type checking, due usage of a `dict` here:

```json
"Padding": {
      "anyOf": [
        {
          "type": "number"
        },
        {
          "additionalProperties": false,
          "properties": {
            "bottom": {
              "type": "number"
            },
            "left": {
              "type": "number"
            },
            "right": {
              "type": "number"
            },
            "top": {
              "type": "number"
            }
          },
          "type": "object"
        }
      ],
      "minimum": 0
    }
```

Reply:
# FIXME

Temporary solution in [`e564ca5` (#3536)](https://github.com/vega/altair/pull/3536/commits/e564ca5a9a163d40669a0e9f1634fe93d16d9ea5)

This is enough for CI passing, but ideally don't want this merged

Reply:
Fixed in [`4933e54` (#3536)](https://github.com/vega/altair/pull/3536/commits/4933e547ae863eb247ea989f5fde37e2dce6019e)